pages/Electric-Dust-Washer.md

```diff
@@ -1,37 +1,15 @@
 
-The Electric Dust Washer is crafted in the [Enhanced Crafting Table](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Enhanced-Crafting-Table). The recipes vary with the machine levels and can be found in the Slimefun guide. The Electric Dust Washer comes in following three tiers:
+## Electric Dust Washer Tiers
 
-![Electric Dust Washer](https://raw.githubusercontent.com/Slimefun/Slimefun-Wiki/master/images/electric-dust-washer-1.png)
+| Tier | Power  | Time   | Buffer | Energy/Ore |
+| :--: | :----: | :----: | :----: | :--------: |
+| I    | 6 J/s  | 4 s    | 128 J  | 24 J       |
+| II   | 10 J/s | 2 s    | 128 J  | 20 J       |
+| III  | 30 J/s | 0.33 s | 512 J  | 10 J       |
 
-![Electric Dust Washer (II)](https://raw.githubusercontent.com/Slimefun/Slimefun-Wiki/master/images/electric-dust-washer-2.png)
+## Tip
 
-![Electric Dust Washer (III)](https://raw.githubusercontent.com/Slimefun/Slimefun-Wiki/master/images/electric-dust-washer-3.png)
-
-## Usage
-
-The Electric Dust Washer has to be connected to an [Electrical Network](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) with sufficient energy to function.  
-The slots of an Electric Dust Washer are as follows:  
-![Electric Dust Washer Slots](https://raw.githubusercontent.com/Slimefun/Slimefun-Wiki/master/images/electric-dust-washer-slots.png)
-- The two cyan glass panes on the left are the input slots where [Sifted Ore](https://github.com/Slimefun/Slimefun4/wiki/Sifted-Ore) can be inserted.
-- The orange slots on the right are the output slots where various [Dusts](https://github.com/Slimefun/Slimefun4/wiki/Dusts) are produced.
-
-Once the output slots are filled, the Electric Dust Washer stops its operation until the output slots are cleared. Therefore, a [Cargo Management System](https://github.com/Slimefun/Slimefun4/wiki/Cargo-Management) is the most efficient way to handle an Electric Dust Washer and can be used to create complex automated dust collection systems.
-
-## Specifications
-
-| Machine Level | Power  | Time   | Energy/Ore |
-| ------------- | ------ | ------ | ---------- |
-| I             | 6 J/s  | 4 s    | 24 J       |
-| II            | 10 J/s | 2 s    | 20 J       |
-| III           | 30 J/s | 0.33 s | 10 J       |
-
-The Level I and Level II machines have a hidden buffer of 128 J as shown below:
-
-![Electric Dust Washer I & II Hidden Buffer](https://raw.githubusercontent.com/Slimefun/Slimefun-Wiki/master/images/electric-dust-washer-1-and-2-buffer.png)
-
-The Level III machine has a hidden buffer of 512 J as shown below:
-
-![Electric Dust Washer III Hidden Buffer](https://raw.githubusercontent.com/Slimefun/Slimefun-Wiki/master/images/electric-dust-washer-3-buffer.png)
\ No newline at end of file
+Seeing as sifted ore can produce many different dusts, it is recommend to hook it up to a cargo system because it only has 2 output slots and it will be full pretty quickly.
```

Comment:
```suggestion
Seeing as sifted ore can produce many different dusts, it is recommended to hook it up to a cargo system because it only has 2 output slots and they will fill pretty quickly.
```
Don't want to be picky but this sounds better imo, also fixed typo in recommended

Reply:
Its not being pick picky, my grammar here is wrong :)